snmp/datadog_checks/snmp/data/default_profiles/brocade-fc-switch.yaml

```diff
@@ -0,0 +1,143 @@
+extends:
+  - brocade.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.1588.2.1.1.*
+  - 1.3.6.1.4.1.1588.2.1.1.32
+  - 1.3.6.1.4.1.1588.2.1.1.44
+  - 1.3.6.1.4.1.1588.2.1.1.62
+  - 1.3.6.1.4.1.1588.2.1.1.64
+  - 1.3.6.1.4.1.1588.2.1.1.66
+  - 1.3.6.1.4.1.1588.2.1.1.71
+  - 1.3.6.1.4.1.1588.2.1.1.72
+  - 1.3.6.1.4.1.1588.2.1.1.77
+  - 1.3.6.1.4.1.1588.2.1.1.83
+  - 1.3.6.1.4.1.1588.2.1.1.109
+  - 1.3.6.1.4.1.1588.2.1.1.117
+  - 1.3.6.1.4.1.1588.2.1.1.118
+  - 1.3.6.1.4.1.1588.2.1.1.120
+  - 1.3.6.1.4.1.1588.2.1.1.133
+  - 1.3.6.1.4.1.1588.2.1.3
+metrics:
+  - MIB: RESOURCE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.1588.2.1.1.1.26.1.0
+      name: cpu.usage
+  - MIB: RESOURCE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.1588.2.1.1.1.26.6.0
+      name: memory.usage
+  - MIB: FIBRE-CHANNEL-FE-MIB
+    table:
+      OID: 1.3.6.1.2.1.75.1.2.1
+      name: fcFxPortStatusTable  # fcFxPortStatusEntry AUGMENTS { fcFxPortEntry }
+    symbols:
+      - name: fcFxPortStatus
+        constant_value_one: true
+    metric_tags:
+      - index: 1  # fcFeModuleIndex is index #1 of fcFxPortEntry
+        tag: fc_fe_module_index
+      - index: 2  # fcFxPortIndex is index #2 of fcFxPortEntry
+        tag: fc_fx_port_index
```

Comment:
@ken-schneider As you can see in the snmprec, the values for this table are like:
```
1.3.6.1.2.1.75.1.2.1.1.1.40140.50904|4|but
1.3.6.1.2.1.75.1.2.1.1.1.48039.17014|4|but
1.3.6.1.2.1.75.1.2.1.1.2.40140.50904|66|515734454
1.3.6.1.2.1.75.1.2.1.1.2.48039.17014|66|3387782298
1.3.6.1.2.1.75.1.2.1.1.3.40140.50904|2|2
1.3.6.1.2.1.75.1.2.1.1.3.48039.17014|2|2
1.3.6.1.2.1.75.1.2.1.1.4.40140.50904|2|2
1.3.6.1.2.1.75.1.2.1.1.4.48039.17014|2|3
```
The index part is `40140.50904` as defined in `fcFxPortEntry` (`fcFxPortStatusEntry AUGMENTS { fcFxPortEntry }`).

Here `index: 1` will lead to tag `fc_fe_module_index` with value `40140`.
And `index: 2` will lead to tag `fc_fx_port_index` with value `50904`.

Reply:
Note also, `constant_value_one: true` as implemented https://github.com/DataDog/datadog-agent/pull/16547/files#diff-5d35532a3a5753998cc6546326b3a20eaa173700b2b26f355a3984e795cb5970R174-R175 

works like this.

1/ If the symbol has `constant_value_one: true`, we collect all the indexes from the `metric_tags` symbols e.g. `40140.50904`.

2/ Use those indexes to creates a metric with value 1 and then add the tags as defined in `metric_tags` section.
Since we have the indexes, the `index: 1` syntax will just extract `40140` from `40140.50904`.

Reply:
See also example of output in e2e test: https://github.com/DataDog/integrations-core/pull/15065/files#diff-00d04c2d06cd1cad7203e010c391fe8111aa655a48185d5931a6ebed1b0392a9R41-R56